docs/configuration.md

```diff
@@ -40,6 +40,7 @@
 | `spark.tispark.jdbc.client_cert_password`       | ``               | The password of `spark.tispark.jdbc.client_cert_store`.                                                                                                                                                                                                                                                               |
 | `spark.tispark.tikv.tls_reload_interval`        | `10s`            | The interval time of checking if there is any reloading certificates. The default is `10s` (10 seconds).                                                                                                                                                                                                              |
 | `spark.tispark.tikv.conn_recycle_time`          | `60s`            | The interval time of cleaning the expired connection with TiKV. Only take effect when enabling cert reloading. The default is `60s` (60 seconds).                                                                                                                                                                     |
+| `spark.tispark.tikv.host_mapping`               | ``               | This is route map used to configure public IP and intranet IP mapping.The format is `{Intranet IP}:{Public IP}`, e.g. `127.0.0.1:192.168.100.100`.                                                                                                                                                                    |
```

Comment:
As for public ip, I recommend using ips like 8.8.8.8. And the example should also show that we use `;` as separator, otherwise, uses can't get it.

Reply:
@zhangyangyu Thanks for your review. I've changed it.